types/react-router-bootstrap/tsconfig.json

```diff
@@ -14,19 +14,17 @@
             "../"
         ],
         "paths": {
```

Comment:
Just remove this section entirely if you're using the newest versions of libraries now.